packages/bundler/src/commands/inline.js

```diff
@@ -113,7 +114,9 @@ async function inlineSassDependencies(
         files: file.path,
         from: REPLACE_REGEX,
         to(_, match) {
-          return `@import '${relativeImportPath}/${match}`;
+          return `@import '${
```

Comment:
Would it help if we used `path.join` here and have the `path` module handle this resolution for us? 👀

Reply:
So I tried using the ``path.join`` function, but it seems to not resolve the separators as expected as shown in the second image below

![Using PathA](https://user-images.githubusercontent.com/16804232/71953979-dd221100-31f4-11ea-87a7-d8e028490945.png)

![Using PathB](https://user-images.githubusercontent.com/16804232/71954088-39853080-31f5-11ea-8fac-735b74553682.png)

Reply:
Am not sure if using ``path.normalize`` may help, but lemme try that as well

Reply:
Nope, changes nothing